models/prices/prices_tokens.sql

```diff
@@ -316,6 +316,8 @@ VALUES
     ("ape-apecoin", "bnb" ,"APE" ,"0x0b079b33b6e72311c6be245f9f660cc385029fc3",18),
     ("metis-metis-token","bnb" ,"Metis" ,"0xe552fb52a4f19e44ef5a967632dbc320b0820639" ,18),
     ("xen-xen-crypto", "bnb", "XEN", "0x2AB0e9e4eE70FFf1fB9D67031E44F6410170d00e", 18),
+    ("bob-bob", "ethereum", "bnb", "0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b", 18),
+
```

Comment:
I am not sure about this line, but "bnb" is at the column `symbol`, maybe this is the targeted change:
```suggestion
    ("bob-bob", "ethereum", "BOB", "0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b", 18),

```
However this change will make this line the same as line 1616:
https://github.com/duneanalytics/spellbook/pull/2373/files#diff-0350e0e6bf4dcce94ae74401c31cb3fe946eaaef8f2b2faa63ad2f4c32d6dbb7R1616

Reply:
Or maybe this:
```suggestion
    ("bob-bob", "bnb", "BOB", "0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b", 18),

```
as it is also the addresses on bsc: https://bscscan.com/address/0xb0b195aefa3650a6908f15cdac7d92f8a5791b0b

Reply:
Good catch thank you!